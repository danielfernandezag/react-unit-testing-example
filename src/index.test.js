import Index from './index';

it('renders app entry point', () => {
	expect(JSON.stringify({}, Index, { _reactInternalInstance: 'censored' })).toMatchSnapshot();
});
