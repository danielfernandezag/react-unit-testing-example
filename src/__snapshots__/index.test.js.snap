// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app entry point 1`] = `"{}"`;
